adam/ontology/phase3_ontology.py

```diff
@@ -197,6 +273,38 @@
     ),
 )
 
+_NULL_ACTION_DESCRIPTION = ActionDescription(
+    frame=ActionDescriptionFrame({AGENT: NULL, THEME: NULL, GOAL: NULL}),
```

Comment:
Rather than using the `NULL` from `asynic.io` (which isn't applicable here) can you make a `_NULL_ACTION_VARIABLE = ActionDescriptionVariable(NULL_NODE)` to use here?